backend/Pipfile

```diff
@@ -45,4 +45,4 @@ typing-extensions = "*"
 drf-excel = "*"
 
 [requires]
-python_version = "3"
```

Comment:
I'm not quite sure why the `Pipfile` and `Pipfile.lock` are changed. In general, if there are no dependency changes, we shouldn't touch these cause they're read directly from our production machines to install. Would suggest following [this](https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file) article to remove these changes from the PR

Reply:
I tried to fix this with the stackoverflow article but it kind of does not look like it did anything per the last commit- I think I could use some help with this at GBM

Reply:
sounds good